cargo-pgrx/src/command/test.rs

```diff
@@ -40,6 +40,12 @@ pub(crate) struct Test {
     /// Don't regenerate the schema
     #[clap(long, short)]
     no_schema: bool,
+    /// Use `sudo` to initialize and run the Postgres test instance as this system user
+    #[clap(long)]
```

Comment:
I don't believe the `#[clap]` macro is correct here, I believe it's supposed to be `#[arg]`?

Reply:
I don't follow?

Reply:
Maybe the docs I was referencing were dated? maybe it's an alias? idk lol